_includes/sidebar.html

```diff
@@ -1,39 +1,53 @@
-      {% if folder.external_url %}
-      <li>
-        <a href="{{folder.external_url}}" target="_blank">{{ folder.title }}</a>
-      </li>
-      {% elsif folder.spacer %}
-      <br>
-      {% else %}
-      <li>
-        <a href="{{folder.url}}">{{ folder.title }}</a>
-
-        {% if folder.folderitems %}
-        <ul style="display:none">
-            {% for folderitem in folder.folderitems %}
-            {% if folderitem.output contains "web" %}
-            {% if folderitem.external_url %}
-            <li><a href="{{folderitem.external_url}}" target="_blank">{{folderitem.title}}</a></li>
-            {% elsif page.url == folderitem.url %}
-            <li class="active"><a href="{{folderitem.url}}">{{folderitem.title}}</a></li>
-            {% else %}
-            <li><a href="{{folderitem.url}}">{{folderitem.title}}</a></li>
-            {% endif %}
-            {% endif %}
-            {% endfor %}
-        </ul>
-        {% endif %}
-      </li>
-      {% endif %}
-
-    {% endif %}
-    {% endfor %}
-    {% endfor %}
+<ul id="mysidebar" class="nav">
+
+  <li class="sidebar-title">Get started</li>
+
+    <ul class="sidebar-items">
+      <li><a href="/setup/">Setup and install</a></li>
+      <li><a href="/getting-started/">Create and run your first app</a></li>
+      <li><a href="/bootstrap-into-dart/">Bootstrap into Dart</a></li>
+      <li><a href="/design-principles/">Design principles</a></li>
+      <li><a href="https://docs.google.com/presentation/d/1cw7A4HbvM_Abv320rVgPVGiUP2msVs7tfGbkgdrTy0I/edit?usp=sharing">Architecture diagrams</a></li>
+    </ul>
+
+  <li class="sidebar-title">Build UIs</li>
+
+    <ul class="sidebar-items">
+      <li><a href="/widgets-intro/">Tour the framework</a></li>
+      <li><a href="/widgets/">Widget and component catalog</a></li>
+      <li><a href="/tutorials/layout/">Build Layouts - Tutorial</a></li>
+      <li><a href="/tutorials/interactive/">Add Interactivity - Tutorial</a></li>
+      <li><a href="/web-analogs/">HTML/CSS patterns in Flutter</a></li>
+      <li><a href="/gestures/">Gestures</a></li>
+      <li><a href="/animations/">Animations</a></li>
+      <li><a href="/custom-fonts/">Custom fonts</a></li>
+      <li><a href="/layout/">Box constraints for layout</a></li>
+      <li><a href="/assets-and-images/">Use assets and images</a></li>
+      <li><a href="/text-input/">Handle text input</a></li>
+      <li><a href="https://www.youtube.com/watch?v=dkyY9WCGMi0">Framework's layered design <i class="fa fa-video-camera" aria-hidden="true"></i></a></li>
+      <li><a href="https://www.youtube.com/watch?v=UUfXWzp0-DU">Flutter's rendering pipeline <i class="fa fa-video-camera" aria-hidden="true"></i></a></li>
+    </ul>
+
+
+  <li class="sidebar-title">Use device and SDK APIs</li>
+
+    <ul class="sidebar-items">
+      <li><a href="/platform-plugins/">Plugins &amp; platform-specific code</a></li>
+      <li><a href="/platform-channels/">Write custom platform-specific code</a></li>
```

Comment:
Maybe the right solution here is to remove the second page from the sidenav? It's already linked from the first page.